.gitignore

```diff
@@ -1,4 +1,5 @@
 build
+!**/dev-bundle/webapp/VAADIN/build
```

Comment:
Why don't ignore all the files inside dev bundle, like this `!**/dev-bundle`? Assets and stats.json have to be committed to VCS as well.

Reply:
The problem here is that in gradle project we have a rule to ignore `build`
If I'm not mistaken, the `!**/dev-bundle` rule would not prevent the `/dev-bundle/webapp/VAADIN/build` from being ignored.
The rule needs to end with the `build` segment

Reply:
You can check it by creating a temp file in `dev-bundle/webapp/VAADIN/build/`, e.g. `touch src/main/dev-bundle/webapp/VAADIN/build/test.me` and the run `git check-ignore -v src/main/dev-bundle/webapp/VAADIN/build/test.me`

With the current rule, the command produces no output, but if we change it to `!**/dev-bundle`, it prints
```
.gitignore:1:build	src/main/dev-bundle/webapp/VAADIN/build/test.me
```